libraries/abstractions/common_io/test/test_iot_uart.c

```diff
@@ -537,7 +537,7 @@ TEST( TEST_IOT_UART, AFQP_IotUARTCancel )
     int32_t lIoctl, lWrite, lClose, lCancel;
     BaseType_t xCallbackReturn;
 
-    uint8_t cSmallBuf[ 2 ] = { 'H', 'I' };
+    uint8_t cSmallBuf[ 3 ] = { 'H', 'I', '\n' };
```

Comment:
nit, you can:

```
uint8_t cSmallBuf[] = "HI"
```

Reply:
Okay, need to keep the \n for the assisted test